package.json

```diff
@@ -9,7 +9,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10"
+    "node": ">=14.0.0"
```

Comment:
I would either remove `engines` altogether or leave it alone to deal with in a separate PR. Bumping to 14 is ... inadequate: technically, nodejs isn't a runtime dependency at all (hence it should be removed), but if we wanted to capture the dev-requirements than the minimum version to compile translations is v16 due to a transitive dep of `formatjs`.